pyaccel/intrabeam_scattering.py

```diff
@@ -0,0 +1,835 @@
+"""."""
```

Comment:
the name of the class already is `IBS`. I thought it would be strange to use `pyaccel.ibs.IBS`, so I decided to use the mode descriptive `intrabeam_scattering` name for the module.